engines/sci/graphics/video32.cpp

```diff
@@ -0,0 +1,393 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "audio/mixer.h"
+#include "common/config-manager.h"
+#include "sci/console.h"
+#include "sci/event.h"
+#include "sci/graphics/cursor.h"
+#include "sci/graphics/frameout.h"
+#include "sci/graphics/palette32.h"
+#include "sci/graphics/text32.h"
+#include "sci/graphics/video32.h"
+#include "sci/sci.h"
+#include "video/coktel_decoder.h"
+
+namespace Sci {
+
+#pragma mark VMDPlayer
+
+VMDPlayer::VMDPlayer(SegManager *segMan, EventManager *eventMan) :
+	_segMan(segMan),
+	_eventMan(eventMan),
+	_decoder(new Video::AdvancedVMDDecoder(Audio::Mixer::kSFXSoundType)),
+
+	_isOpen(false),
+	_isInitialized(false),
+	_yieldInterval(0),
+	_lastYieldedFrameNo(0),
+
+	_plane(nullptr),
+	_screenItem(nullptr),
+	_planeIsOwned(true),
+	_priority(0),
+	_doublePixels(false),
+	_stretchVertical(false),
+	_blackLines(false),
+	_leaveScreenBlack(false),
+	_leaveLastFrame(false),
+
+	_blackoutPlane(nullptr),
+
+	_startColor(0),
+	_endColor(255),
+	_blackPalette(false),
+
+	_boostPercent(100),
+	_boostStartColor(0),
+	_boostEndColor(255),
+
+	 _showCursor(false) {}
```

Comment:
Good catch. Xcode invisibles are [broken](http://stackoverflow.com/questions/9244249/how-can-i-change-the-color-of-invisibles-in-xcode4) and so sometimes things like this sneak in despite my best efforts. :\